### DISCUSSION TITLE
MiniBoard & Board Questions

### DISCUSSION THREAD

**Discussion by meturgut** [rexrainbow/phaser3-rex-notes#493] [Q&A]
Hello @rexrainbow ,

I’m using the Board and MiniBoard classes for my grid game.

1-When two MiniBoards overlap and both are set to draggable, dragging one moves both together. How can I make it so only one of them is dragged?
2-I need to access the position information of the GameObjects inside the MiniBoard. How can I achieve this? Should I use a `getAll` function for this?
3-I want to use a `getBounds()` method like in a `Container`, but in the MiniBoard, the width and height return 0 even if there are objects inside. How can I fix this?
4-I noticed in the example [here](https://codepen.io/rexrainbow/pen/PoeZOGo) that functions like `var bounds = board.getBoardBounds(true); `and similar ones do not appear in TypeScript, and the VS Code editor shows compile errors. However, the game runs without any issues. How can I prevent this?

Kind regards,

**Comment by rexrainbow** [comment]
Most of them are feature requests. Will reply at #494

**Comment by rexrainbow** [comment]
Responded [here](https://github.com/rexrainbow/phaser3-rex-notes/issues/494#issuecomment-2508973610)
